tumbleweed

```diff
@@ -161,6 +161,11 @@ tumbleweed_switch()
     echo "choosing latest version"
   fi
 
+  if [ -z "$version" ] ; then
+    echo "There is no version to upgrade to. Please check your internet connection"
```

Comment:
Perhaps, `Unable to determine latest version (likely due to network connection issue)` as `There is no version to upgrade to` is not necessarily true, but the script cannot determine if that is the case.

Reply:
That is true, that message is more concise.